packages/MarkdownEditor-Core.package/MarkdownDelimiterEmphasis.class/instance/attributeStartingAt..st

```diff
@@ -0,0 +1,10 @@
+attributes
+attributeStartingAt: aMarkdownDelimiterEmphasis
+
+	| attrLen start end attributeClass |
+
+	attrLen := {self length. aMarkdownDelimiterEmphasis length. self boldLength} min.
+	start := aMarkdownDelimiterEmphasis endPosition - attrLen + 1.
```

Comment:
Also, this index shifting is driving me crazy... Are attributes inclusive and tokens not or are things just like this is written?

Reply:
As far as I can remember everything is inclusive. It's simply a bit complicated because we want the tokens to be included as bold or italic. But if we have an opener that is 5 stars long and it opens a bold section then we only want its last two stars to be a part of that bold section. For that reason we need to turn an inclusive ending to an inclusive start which requires a +1 sadly...